src/style.css

```diff
@@ -1 +1,78 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
```

Comment:
Consider using `font-display: swap;` in the `@font-face` rule to improve loading performance and provide a better user experience by avoiding invisible text during font loading.